endpoints/cookie_sync.go

```diff
@@ -530,5 +530,5 @@ func (p usersyncPrivacy) CCPAAllowsBidderSync(bidder string) bool {
 func (p usersyncPrivacy) ActivityAllowsUserSync(bidder string) bool {
 	activityResult := p.activityControl.Evaluate(privacy.ActivitySyncUser,
 		privacy.ScopedName{Scope: privacy.ScopeTypeBidder, Name: bidder})
-	return activityResult == privacy.ActivityAllow
+	return activityResult != privacy.ActivityDeny
```

Comment:
I think we are still leaving ourselves open to the issue recurring. I would suggest:

- Change  `func (e ActivityControl) Evaluate(activity Activity, target ScopedName) ActivityResult` to `func (e ActivityControl) Allow(activity Activity, target ScopedName) bool`. The boolean result will ensure we're not messing up downstream.

- Per https://github.com/prebid/prebid-server/issues/2591, `If default is unspecified, the hardcoded default-default is true`. To better implement, `if !planDefined` should return `true`. We can also change `ActivityPlan.defaultResult` to a bool.